Packs/CheckPointHarmonyEndpoint/Integrations/CheckPointHarmonyEndpoint/CheckPointHarmonyEndpoint.yml

```diff
@@ -0,0 +1,3217 @@
+      description: Determines whether to inform the user, via a UserCheck (popup) message, that the operation is taking place.
+      auto: PREDEFINED
+      defaultValue: 'true'
+      predefined:
+      - 'true'
+      - 'false'
+    - name: allow_postpone
+      description: Determines whether to allow the user to postpone the operation.
+      auto: PREDEFINED
+      defaultValue: 'true'
+      predefined:
+      - 'true'
+      - 'false'
+    - name: page
+      description: 'Page number of paginated results. Minimum value: 1.'
+    - name: page_size
+      description: The number of items per page.
+    - name: limit
+      description: The maximum number of records to retrieve.
+      defaultValue: '50'
+    - name: interval
+      description: The interval between each poll in seconds. Minimum value is `10`. 
+      defaultValue: '30'
+    - name: timeout
+      description: The timeout for the polling in seconds.
+      defaultValue: '600'
+    - name: job_id
+      description: The job ID to fetch data for. Hidden argument.
+      hidden: true
+    polling: true
+    outputs:
+    - contextPath: HarmonyEP.RegistryKeyAdd.PushOperation.job_id
+      description: The job ID of the remediation operation.
+      type: String
+    - contextPath: HarmonyEP.RegistryKeyAdd.PushOperation.id
+      description: The remediation operation ID.
+      type: String
+    - contextPath: HarmonyEP.RegistryKeyAdd.PushOperation.status
+      description: Describes possible states in which a push operation may be in regards to a specific device.
+      type: String
+    - contextPath: HarmonyEP.RegistryKeyAdd.PushOperation.response.status
+      description: Push operation response status.
+      type: String
+    - contextPath: HarmonyEP.RegistryKeyAdd.PushOperation.response.output
+      description: Push operation response output.
+      type: String
+    - contextPath: HarmonyEP.RegistryKeyAdd.PushOperation.machine.ipAddress
+      description: The client device's IPv4 address.
+      type: String
+    - contextPath: HarmonyEP.RegistryKeyAdd.PushOperation.machine.name
+      description: The client device's name.
+      type: String
+    - contextPath: HarmonyEP.RegistryKeyAdd.PushOperation.machine.id
+      description: The client device's unique ID.
+      type: String
+  - name: harmony-ep-agent-registry-key-delete
+    description: "Removes the given registry key or value to the registry of computers matching the given query. Note that you must specify at least one of the following filter arguments: computer_ids, computer_names, computer_ips, computer_group_names, computer_types, computer_deployment_status, computer_last_connection, or filter. "
+    arguments:
+    - name: is_redirected
+      description: Determines if the key should reside under WOW6432Node. Keys intended for 64bit versions of Windows may target 32bit versions by using setting 'true' here. Thus specifying that the registry key/value be removed under the WOW6432Node.
```

Comment:
Should this be:
 Determines if the key should be removed from under WOW6432Node. Keys intended for 64bit versions of Windows may target 32bit versions by setting this value to 'true', thus specifying that the registry key/value be removed under the WOW6432Node.